contracts/interchain-token-service/tests/interchain_transfer.rs

```diff
@@ -63,3 +64,59 @@ fn interchain_transfer_send_fails_on_insufficient_balance() {
         &gas_token,
     );
 }
+
+#[test]
+fn interchain_transfer_fails_zero_amount() {
```

Comment:
```suggestion
fn interchain_transfer_fails_on_zero_amount() {
```
fine to make the test names more like a sentence for readability